hack/charts/installer/rolebased/templates/operator-installer.yaml

```diff
@@ -79,6 +79,26 @@ rules:
   - get
   - update
   - patch
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - hostingautoscalingpolicies
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - hostingautoscalingpolicies/status
+  verbs:
+  - get
+  - patch
+  - update
```

Comment:
Can we have a make target to copy these values over from the other installer? This seems like it's easily automated?